Formula/ispc.rb

```diff
@@ -17,9 +18,16 @@ class Ispc < Formula
   depends_on "llvm@4"
 
   def install
+    # The standard include paths for clang supplied by the llvm@4 formula do not include
+    # C headers such as unistd.h. Add the path to those headers explicitly so that
+    # generation of the ispc builtins and standard library do not silently fail.
+    inreplace "cmake/GenerateBuiltins.cmake", "${CLANG_EXECUTABLE}",
+      "${CLANG_EXECUTABLE} -I#{MacOS.sdk_path}/usr/include"
+
     args = std_cmake_args + %W[
       -DISPC_INCLUDE_EXAMPLES=OFF
       -DISPC_INCLUDE_TESTS=OFF
+      -DISPC_INCLUDE_UTILS=OFF
```

Comment:
Seems orthogonal to the issue. What motivated this change?

Reply:
Parity with the previous version of the formula, i.e., an omission when the formula switched build systems.